src/goto-programs/remove_asm.cpp

```diff
@@ -25,9 +25,7 @@ Date:   December 2014
 class remove_asmt
 {
 public:
-  remove_asmt(
-    symbol_tablet &_symbol_table,
-    goto_functionst &_goto_functions)
+  remove_asmt(symbol_tablet &_symbol_table, goto_functionst &_goto_functions)
```

Comment:
Nit picking, but should this change possibly go in the previous commit?

Reply:
Yes, will now happen after merging this, then rebasing #2714